### PR TITLE
Add branding assets for Jadu Deployer, and new jadu purple colour palette

### LINKED ISSUE #1024: Add Jadu Deployer colour palette/logo/favicon/integrations logo
**Is your feature request related to a problem? Please describe.**
Jadu Deployer has a new colour palette and logo which currently are not available in the Pulsar library

**Describe the solution you'd like**
- Add a new palette branding with the following colours:

```
                darkest:     rgb(46, 54, 112),
                dark:        rgb(61, 71, 150),
```
- Add the Jadu Deployer logo branding with favicon as well
- An integration logo for Deployer as well

**Describe alternatives you've considered**
- Could do for now with only favicon and integration logo, even if the work isn't yet merged into Pulsar since we can add/override stylings in the project

**Additional context**
- Added screenshots and the .svg with the logo

deployer-icon

[deployer-logo.svg.zip](https://github.com/jadu/pulsar/files/3656910/deployer-logo.svg.zip)

deployer-dashboard

deployer-login

Thank you!

### PR DESCRIPTION
**Pull request by Stanton**
* Add deployer icon source SVG
* Create white-on-transparent and black-on-transparent logo variations
* Add `grunt realFavicon:deployer` task to generate favicons
* Add purple colour variations as per brand guidelines

**Favicons**
All generated favicon are in `/images/favicons/deployer`, some icons require a background colour (like when a webpage is saved as an app icon in iOS), this has been set to white-on-purple.

android-chrome-512x512

**Files of note:**
`/views/pulsar/components/favicons-deployer.html` (include this in the `<head>`)
`/images/branding/logo-deployer-black.svg`
`/images/branding/logo-deployer-white.svg`

**Colour palette**
Screenshot 2019-09-26 at 16 27 18

**Integration icon**
Integration icons aren't bundled within the Pulsar repository, and should be created from either the source branding images, or copied from an appropriately large favicon (and stored along with the actual integration codebase).

Closes #1024

### CODE REVIEW

File: stylesheets/_palette.base.scss [hunk 1]
```diff
@@ -9,6 +9,11 @@
 \* -------------------------------------------------------------------------- */
 
 $palette-branding: (
+    jadu-purple: (
+        darkest:    rgb(46, 54, 112),
+        dark:       rgb(51, 60, 121),
+        base:       rgb(61, 71, 150)
+    ),
```
Comment (skmetaly) [inline stylesheets/_palette.base.scss]: do we need a light and pale versions as well?

Comment (Stanton) [reply stylesheets/_palette.base.scss]: Apologies, I'd done them after Lee sent me the branding guidelines but not committed them 🤦‍♂

**Comment by skmetaly** [comment]
Hi @Stanton ,

Aren't we using the white-on-purple background for the main favicon as well? Asking cause when the tab is selected, you can't see it at all.

deployer_favicon
deployer_favicon_selected

**Comment by Stanton** [comment]
I'll check today when I get in

**Comment by Stanton** [comment]
@skmetaly Small favicon should now use white-on-purple scheme.

favicon-16x16

**Comment by skmetaly** [comment]
thanks @Stanton. Looks very good now

**Pull request merged**